rebar.config

```diff
@@ -1,4 +1,4 @@
 {erl_opts, [debug_info]}.
 
-{deps, [ yamerl, jsx,
```

Comment:
jsx is explicitly used in at least swagger_generate so direct dependency seems more readable.